### DISCUSSION TITLE
Why doesn't httpx support omitting None params?

### DISCUSSION THREAD

**Discussion by MarcoRocco** [projectdiscovery/httpx#306] [Q&A] [answered]
As mentioned in the documentation, httpx doesn't support omitting params which are None, unlike the `requests` library. What is the reasoning behind this and are there any plans to support this in future releases?

**Comment by ehsandeep** [comment, marked as answer]
@MarcoRocco this might be related to https://github.com/encode/httpx?

**Reply by MarcoRocco** [reply]
Yes, that's embarrassing, sorry.

**Reply by ehsandeep** [reply]
np :)
